test/test_helpers.py

```diff
@@ -186,7 +186,7 @@ def test_replace_fill_values_invalid(self):
         """Tests _replace_fill_values() helper function attempting to use a
         fill value that is not representable by the current dtype."""
 
-        face_nodes = np.array([[1, 2, -1], [-1, -1, -1]], dtype=np.uint32)
+        face_nodes = np.array([[1, 2, -1], [-1, -1, -1]], dtype=np.int32)
```

Comment:
was this causing issues with `uint32`?

Reply:
Yeah, which makes sense (`-1` can't be represented in `uint32`). Looks like NumPy 2.0 correctly handles this now.